static/css/components/mybooks-menu.less

```diff
@@ -4,6 +4,59 @@
  *
  * http://localhost:8080/account/books/want-to-read
  */
+
+/* 'create a new list' within sidebar */
+.sidebar-section {
+  a.dropclick {
+    display: none;
+  }
+
+  .create {
+    text-align: center;
+  }
+
+  .create-new-list {
```

Comment:
I'm not seeing this anywhere in the html... is a piece missing perhaps?

Reply:
The code:
$ user_lists = []
$:macros.ReadingLogDropper(user_lists)
was added to sidebar.html to create the alink with the classname “create-new-list”
This css targets this classname when it’s within the sidebar-section in sidebar.html

Reply:
This css was modified for the CreateList macro which has the button.